contrib/asmap/README.md

```diff
@@ -11,3 +11,42 @@ python3 asmap-tool.py decode /path/to/input.file /path/to/output.file
 python3 asmap-tool.py diff /path/to/first.file /path/to/second.file
 python3 asmap-tool.py diff-addrs /path/to/first.file /path/to/second.file addrs.file
 ```
+
+### Comparing ASmaps
+
+AS control of IP networks changes frequently, therefore it can be useful to get
+the changes between to ASmaps via the `diff` and `diff_addrs` commands.
+
+`diff` takes two ASmap files, and returns a file detailing the changes
```

Comment:
75b145eef57053f24bc58531163408271d17eb59: nit: I think it would be great to mention that `diff` may take a while to execute. In my last try it took around 2 minutes and doesn't show anything until the end.